config/crd/bases/infrastructure.cluster.x-k8s.io_azuremanagedcontrolplanes.yaml

```diff
@@ -777,103 +778,198 @@ spec:
                       name:
                         type: string
                       privateEndpoints:
-                        description: PrivateEndpoints is a slice of Virtual Network
-                          private endpoints to create for the subnets.
-                        items:
-                          description: PrivateEndpointSpec configures an Azure Private
-                            Endpoint.
-                          properties:
-                            applicationSecurityGroups:
-                              description: ApplicationSecurityGroups specifies the
-                                Application security group in which the private endpoint
-                                IP configuration is included.
-                              items:
+                        allOf:
```

Comment:
Not sure why the `allOf` pops up when adding a separate `ManagedControlPlaneVirtualNetworkClassSpec` type.. This is causing the CRD to be invalid. Will be looking into this.